MWWeb/MWWeb/SceneDelegate.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  SceneDelegate.swift
+//  MWWeb
+//
+//  Created by Xavi Moll on 23/12/20.
+//  Copyright © 2020 Future Workshops. All rights reserved.
+//
+
+import Foundation
+import MWWebPlugin
+import MobileWorkflowCore
+
+class SceneDelegate: UIResponder, UIWindowSceneDelegate {
+
+    var window: UIWindow?
+    private var urlSchemeManagers: [URLSchemeManager] = []
+    private var rootViewController: MobileWorkflowRootViewController!
+    
+    private var appDelegate: AppDelegate? { UIApplication.shared.delegate as? AppDelegate }
+    
+    func scene(_ scene: UIScene, willConnectTo session: UISceneSession, options connectionOptions: UIScene.ConnectionOptions) {
+        guard let windowScene = scene as? UIWindowScene else { return }
+        
+        let eventService = EventServiceImplementation()
+        self.appDelegate?.eventDelegate = eventService
+        
+        let manager = AppConfigurationManager(
+            withPlugins: [MWWebPlugin.self],
+            fileManager: FileManager.default,
+            networkService: NetworkAsyncTaskService(),
+            eventService: eventService,
+            supportServices: [AuthenticationService(credentialsStore: CredentialsStore(), authRedirectHandler: eventService.authRedirectHandler())]
+        )
```

Comment:
@igorcferreira Can you doublecheck that this is okay? I went to copy this from a previous Plugin but with the changes that were merged on Core a few days ago, the network service and authRedirector had changed. This code was inspired from looking at this: https://github.com/FutureWorkshops/MobileWorkflow-iOS/pull/190/files#diff-7e409423f82582dfe563ca19675181129973eaa029fc0da43a82bca29932c0b3R34